### DISCUSSION TITLE
Customization

### DISCUSSION THREAD

**Discussion by mogosam** [sunilsharma08/XNLogger#59] [Ideas]
Is it possible to be able to override the presenting view controller so it doesn't pop from window and be able to customize the colours, etc?

**Comment by sunilsharma08** [comment]
Currently, you can present it the way you want for example
```swift
if let tabbarVC = UIStoryboard(name: "XNUIMain", bundle: Bundle(for: XNUIManager.self)).instantiateViewController(withIdentifier: "xnlMainTabBarController") as? UITabBarController {
        tabbarVC.modalPresentationStyle = .overFullScreen
        self.present(tabbarVC, animated: true, completion: nil)
 }
```

But customisation and other events may not work as expected.

Adding this feature to lib with better support is a cool idea. I will try to add this feature when I will have time or If you are interested in adding this feature, please raise PR to `develop` branch.
